src/component/selection/getRangeClientRects.js

```diff
@@ -55,6 +56,7 @@ function getRangeClientRectsChrome(range: Range): Array<ClientRect> {
     'Found an unexpected detached subtree when getting range client rects.'
   );
 }
+/* eslint-enable consistent-return */
```

Comment:
Is this because of the invariant at the end? Should we just add a `return [];` at the bottom instead?